src/backends/kubernetes/kubernetes_backend.py

```diff
@@ -415,6 +420,37 @@ def create_bitcoind_container(self, tank) -> client.V1Container:
         )
         return bitcoind_container
 
+    def create_prometheus_container(self, tank) -> client.V1Container:
+        return client.V1Container(
+            name="prometheus",
+            image="jvstein/bitcoin-prometheus-exporter:latest",
+            env=[
+                client.V1EnvVar(name="BITCOIN_RPC_HOST", value="localhost"),
+                client.V1EnvVar(name="BITCOIN_RPC_PORT", value=str(tank.rpc_port)),
+                client.V1EnvVar(name="BITCOIN_RPC_USER", value=tank.rpc_user),
+                client.V1EnvVar(name="BITCOIN_RPC_PASSWORD", value=tank.rpc_password),
+            ],
+        )
+
+    def apply_prometheus_scrape_configs(self, tanks):
+        for tank in tanks:
+            if not tank.exporter:
+                continue
+
+            bitcoind_service_name = self.get_service_name(tank.index)
+            target = f"{bitcoind_service_name}.{self.namespace}:9332"
+            scrape_config = {
+                "apiVersion": "monitoring.coreos.com/v1alpha1",
+                "kind": "ScrapeConfig",
+                "metadata": {"name": "test", "namespace": "prometheus"},
+                "spec": {"scrapeInterval": "5s", "staticConfigs": [{"targets": [target]}]},
```

Comment:
in a4d4ee1319cd135f519210a2be75924cd05dc5cd "Add prometheus to logging stack":
perhaps (in the future) this `5s` constant should be configurable in the graph? Unclear to me currently.

Reply:
Scrapeconfig is no longer used, bad (no) squashing on my part.